frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -279,13 +279,20 @@ function* splitOrMergeAgglomerate(action: MergeTreesAction | DeleteEdgeAction) {
   const isHdf5MappingEnabled = yield* call(ensureHdf5MappingIsEnabled, layerName);
   if (!isHdf5MappingEnabled) return;
 
+  const busyBlockingInfo = yield* select((state) => state.uiInformation.busyBlockingInfo);
+  if (busyBlockingInfo.isBusy) {
+    console.warn(`Ignoring proofreading action (reason: ${busyBlockingInfo.reason || "null"})`);
+    return;
+  }
+
+  yield* put(setBusyBlockingInfoAction(true, "Proofreading action"));
+
   if (!volumeTracing.mappingIsEditable) {
     try {
       yield* call(createEditableMapping);
     } catch (e) {
       console.error(e);
-      // Undo the last splitting/merging action since the proofreading action did not succeed
-      yield* put(undoAction());
```

Comment:
This was an oversight while merging the original proofreading PR. Undo/Redo is actually not allowed if the proofreading tool is active and a warning in that regard is shown to the user if it is triggered. This was added kind of last-minute, because the actual proofreading actions cannot be undone yet. Unfortunately, that's why this line had to be removed for now.